# Notes
## Challenge: fixme1.py

Running the given python file gives an error.
### Command:
```bash
python3 fixme1.py
```
### Output:
```
  File "fixme1.py", line 20
    print('That is correct! Here\'s your flag: ' + flag)
    ^
IndentationError: unexpected indent
```
After fixing the IndentationError, the code runs and gives the flag.
### Command:
```bash
python3 fixme1.py
```
### Output:
```
That is correct! Here's your flag: picoCTF{1nd3nt1ty_cr1515_6a476c8f}
```
